openapi/multi-yaml/paths/checkouts.yaml

```diff
@@ -56,6 +56,19 @@ post:
                       type: number
                       description: custom application fee amount that applies to bank account payment method.
                       example: 150
+            payment_overrides:
```

Comment:
I'm not sure about the param `payment_overrides` parameter name. While it does indicate that it overrides values, is that level of specificity necessary?

For instance, application_fees also overrides configured fees but doesn’t include an `_overrides` suffix in its name.

Reply:
Ok, changed